package.json

```diff
@@ -1,75 +1,44 @@
 {
-  "name": "coda",
-  "version": "1.0.0",
-  "description": "California Oak Disease and Arthropod Database",
-  "main": "/src/index.jsx",
+  "name": "coda-client-v2-temp",
```

Comment:
could probably restore the head matter (coda, 1.0.0, description field, etc)

Reply:
Done